data/processed/test_data.csv

```diff
@@ -1,886 +1,886 @@
 ,Marital status,Application mode,Application order,Course,Daytime/evening attendance,Previous qualification,Previous qualification (grade),Nacionality,Mother qualification,Father qualification,Mother occupation,Father occupation,Admission grade,Displaced,Educational special needs,Debtor,Tuition fees up to date,Gender,Scholarship holder,Age at enrollment,International,Curricular units 1st sem (credited),Curricular units 1st sem (enrolled),Curricular units 1st sem (evaluations),Curricular units 1st sem (approved),Curricular units 1st sem (grade),Curricular units 1st sem (without evaluations),Curricular units 2nd sem (credited),Curricular units 2nd sem (enrolled),Curricular units 2nd sem (evaluations),Curricular units 2nd sem (approved),Curricular units 2nd sem (grade),Curricular units 2nd sem (without evaluations),Unemployment rate,Inflation rate,GDP,Target
-3781,1,1,3,9070,1,1,139.0,1,1,38,4,7,142.2,1,0,0,1,0,1,19,0,0,6,6,6,12.833333333333334,0,0,6,6,6,12.833333333333334,0,9.4,-0.8,-3.12,Graduate
```

Comment:
Did we update the script anywhere? I saw the Target column is not using "Graduate", "Dropout" or "Enrolled" anymore, it's using 0,1,2 now